content/general/authentication.textile

```diff
@@ -84,26 +86,26 @@ Whilst Basic Authentication is simple, we recommend it to be only used on the se
-* an "<span lang="default">@authUrl@</span><span lang="csharp">@AuthUrl@</span>":/realtime/types#client-options or "<span lang="default">@authCallback@</span><span lang="csharp">@AuthCallback@</span>":/realtime/types#client-options is provided that returns a token or token request;
+* an "<span lang="default">@authUrl@</span><span lang="csharp">@AuthUrl@</span>":/realtime/types#client-options or "<span lang="default">@authCallback@</span><span lang="csharp">@AuthCallback@</span>":/realtime/types#client-options is provided that returns an Ably-compatible token or Ably TokenRequest;
 * a "<span lang="default">@clientId@</span><span lang="csharp">@ClientId@</span>":/realtime/types#client-options is provided;
 * "<span lang="default">@useTokenAuth@</span><span lang="csharp">@UseTokenAuth@</span>":/realtime/types#client-options is true
 
-Token authentication is typically done in one of two ways:
+Token authentication is typically done in one of four ways:
 
-h4. Signed token request is created by your servers and passed to clients
+h4(#token-request-process). Ably TokenRequest is created by your servers and passed to clients
 
-Using our REST or Realtime client libraries, a "signed token request is generated from your servers":/realtime/authentication#create-token-request and handed to the client-side client library. The client-side client library then uses that "signed token request":/realtime/types#token-request to "request a token":/realtime/authentication#request-token from Ably and subsequently authenticate using that token. This is the recommended approach for authentication as: a "signed token request":/realtime/types#token-request can be generated securely by your servers without communicating with Ably; your secret API key is never shared with Ably or your clients. Signed token requests cannot be tampered with, must be used soon after creation and can only be used once. This process is depicted in the following diagram:
+Using our REST or Realtime client libraries, a "Ably TokenRequest is generated from your servers":/realtime/authentication#create-token-request and handed to the client-side client library. The client-side client library then uses that "Ably TokenRequest":/realtime/types#token-request to "request an Ably Token":/realtime/authentication#request-token from Ably and subsequently authenticate using that Ably Token. This is the recommended approach for authentication as: a "Ably TokenRequest":/realtime/types#token-request can be generated securely by your servers without communicating with Ably; your secret API key is never shared with Ably or your clients. Ably TokenRequests cannot be tampered with due to being signed, must be used soon after creation and can only be used once. This process is depicted in the following diagram:
```

Comment:
Should be "an Ably TokenRequest", not a Ably TokenRequest. As above, should be code styled, I won't mention that again and will assume you will change everywhere.  Also, might be worth checking with @paddybyers, but these are signed TokenRequests, which I realise one could assume all token requests are, but they are not.  So I think we should be consistent with that in that a signed token request can be trusted by the Ably platform from an untrusted and unauthenticated client.